com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/script/dialog/TransFileBySSHDialog.java

```diff
@@ -302,6 +307,59 @@ public void widgetSelected(SelectionEvent ev) {
         btnSaveHost.setText(Messages.btnSaveRemoteServer);
         btnSaveHost.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false, 2, 1));
         btnSaveHost.setSelection(true);
+
+        Label lblChangeMigrationScriptDir = new Label(result, SWT.NONE);
+        lblChangeMigrationScriptDir.setText(Messages.lblScriptChangeDirectory);
+
+        Group directoryGrp = new Group(result, SWT.NONE);
+        directoryGrp.setLayout(new GridLayout(4, false));
+        directoryGrp.setLayoutData(new GridData(SWT.FILL, SWT.TOP, true, false));
+
+        Image questionImage = Display.getDefault().getSystemImage(SWT.ICON_QUESTION);
+        final int width = questionImage.getBounds().width;
+        final int height = questionImage.getBounds().height;
+        Image questionImage050 =
+                new Image(
+                        Display.getDefault(),
+                        questionImage
+                                .getImageData()
+                                .scaledTo((int) (width * 0.5), (int) (height * 0.5)));
+
+        Label lblSourceJdbc = new Label(directoryGrp, SWT.NONE);
+        lblSourceJdbc.setText(Messages.lblSourceJdbcDriverDir);
+        lblSourceJdbc.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
+
+        Label lblSourceJdbcQuestion = new Label(directoryGrp, SWT.NONE);
+        lblSourceJdbcQuestion.setImage(questionImage050);
+        lblSourceJdbcQuestion.setToolTipText(Messages.lblSourceJdbcDriverDirQuestion);
+        lblSourceJdbcQuestion.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
+
+        txtSourceJdbcDriverDir = new Text(directoryGrp, SWT.BORDER);
+        txtSourceJdbcDriverDir.setLayoutData(new GridData(SWT.FILL, SWT.CENTER, true, false, 2, 1));
+
+        Label lblTargetJdbc = new Label(directoryGrp, SWT.NONE);
+        lblTargetJdbc.setText(Messages.lblTargetJdbcDriverDir);
+        lblTargetJdbc.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
+
+        Label lblTargetJdbcQuestion = new Label(directoryGrp, SWT.NONE);
+        lblTargetJdbcQuestion.setImage(questionImage050);
+        lblTargetJdbcQuestion.setToolTipText(Messages.lblTargetJdbcDriverDirQuestion);
+        lblTargetJdbcQuestion.setLayoutData(new GridData(SWT.LEFT, SWT.CENTER, false, false));
+
+        txtTargetJdbcDriverDir = new Text(directoryGrp, SWT.BORDER);
```

Comment:
Wouldn't it be good to have a default value(already exists path)?

Reply:
I think it's easier for users to use.
Thank you for the idea.